From 8fc6844529e1b9559e04ddc4be0c80aaa1aa3678 Mon Sep 17 00:00:00 2001
From: Joshua Strobl <joshua@streambits.io>
Date: Wed, 25 Mar 2020 22:15:34 +0200
Subject: [PATCH 1/1] Revert "Gtk-3.2x&4.0: [Budgie-desktop] Shrink the Raven's
 sound widget placeholder label more"

This reverts commit 63fd9b06cb353588d184ae57a072f034f8630c6f. It looks bad.
---
 src/gtk/sass/common/_3rd-party.scss | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gtk/sass/common/_3rd-party.scss b/src/gtk/sass/common/_3rd-party.scss
index 8588473..9196ad7 100644
--- a/src/gtk/sass/common/_3rd-party.scss
+++ b/src/gtk/sass/common/_3rd-party.scss
@@ -687,7 +687,7 @@ $switcher_height: $widget_size - if($ref_weight < 1.0, 10.7px, 5.3px);
       // placehoder label
       widget.raven-background stack > box > revealer > box.vertical > label {
         @extend %dim_label;
-        font-size: 80%;
+        font-size: 90%;
       }
 
       list { // apps/devices list
-- 
2.26.0

